### ISSUE TITLE
Cannot find module 'trpc-sveltekit/websocket' or its corresponding type declarations.

### ISSUE THREAD

**Issue by neutron0831** [icflorescu/trpc-sveltekit#95]
**Describe the bug**
Error occurs when importing `trpc-sveltekit/websocket`.
```sh
Cannot find module 'trpc-sveltekit/websocket' or its corresponding type declarations.
```

**To Reproduce**
1. Create a project and install `trpc-sveltekit`
```sh
$ pnpm create svelte@latest my-app
$ pnpm add trpc-sveltekit @trpc/server @trpc/client
```
2. Import 'trpc-sveltekit/websocket' as described in the WebSocket example
```ts
import { SvelteKitTRPCWSServer } from 'trpc-sveltekit/websocket'
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^
SvelteKitTRPCWSServer(import.meta.url)
```

**Expected behavior**
`trpc-sveltekit/dist/websocket/*.d.ts` files are imported without error.

**Screenshots**
![screenshot](https://github.com/icflorescu/trpc-sveltekit/assets/19346544/0b585bdb-40b4-4404-805b-830492ee72ec)

**Additional context**
The application works fine with `pnpm dev`, so the modules are imported without any problem. But the definition files are not imported, so type inference does not work.
On the other hand, when I import `trpc-sveltekit/dist/websocket`, the definition files are imported and the error disappears. But instead the following error occurs and the modules are not imported.
```sh
$ pnpm start
node:internal/errors:491
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/websocket' is not defined by "exports" in ~/my-app/node_modules/trpc-sveltekit/package.json imported from ~/my-app/wsServer.js
    at new NodeError (node:internal/errors:400:5)
    at exportsNotFound (node:internal/modules/esm/resolve:361:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:697:9)
    at packageResolve (node:internal/modules/esm/resolve:872:14)
    at moduleResolve (node:internal/modules/esm/resolve:938:20)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:842:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

Node.js v18.13.0
```

**Comment by neutron0831** [comment]
I found that the error did not appear until v3.5.18 and started appearing from v3.5.19.
So I believe the cause lies somewhere in the changes between those two versions.
https://github.com/icflorescu/trpc-sveltekit/compare/dce02fe82e0422973aca0f638bb8e575945ca694...12a29020aa7abda25c970442287e92db27521deb

**Comment by neutron0831** [comment]
I confirmed that the error disappears after reverting the following commit.
https://github.com/icflorescu/trpc-sveltekit/commit/531358c3fdc3c156848c18764437ca65aad20213
However, I have never published an npm package myself, so I don't know what the correct way to handle this issue.
If anyone knows anything I would appreciate your information:pray:

**Comment by neutron0831** [comment]
> I found that the error did not appear until v3.5.18 and started appearing from v3.5.19.
> So I believe the cause lies somewhere in the changes between those two versions.
> [dce02fe...12a2902](https://github.com/icflorescu/trpc-sveltekit/compare/dce02fe82e0422973aca0f638bb8e575945ca694...12a29020aa7abda25c970442287e92db27521deb)

The only solution I could find is to manually append the `typesVersions` (removed by https://github.com/icflorescu/trpc-sveltekit/commit/531358c3fdc3c156848c18764437ca65aad20213) to `package.json`.

```diff
+  "typesVersions": {
+    "*": {
+      "websocket": [
+        "dist/websocket/index.d.ts"
+      ]
+    }
+  },
```

**Comment by icflorescu** [comment]
Should be fixed in the latest version (`3.5.24`).
